lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -114,6 +114,18 @@ def skip_pch?(specs)
             specs.any? { |spec| spec.prefix_header_file.is_a?(FalseClass) }
           end
 
+          # True if info.plist generation should be skipped
+          #
+          # @param [PXNativeTarget] native_target
+          #
+          # @return [Boolean] Whether the target should build an Info.plist file
+          #
+          def skip_info_plist?(native_target)
+            return true if target.static_framework?
+            existing_setting = native_target.resolved_build_setting('INFOPLIST_FILE', true).values.compact
```

Comment:
I guess passing `true` here makes sense since we know the `.xcconfig` has been created and assigned there and its within our control.